llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -2756,6 +2756,147 @@ LogicalResult WinogradOutputTransformOp::reifyResultShapes(
       .reifyResultShapes(b, reifiedReturnShapes);
 }
 
+//===----------------------------------------------------------------------===//
+// FlashAttentionFwdOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult FlashAttentionFwdOp::verify() {
+  Operation *op = getOperation();
+  if (getNumInputs() != 3) {
+    return op->emitOpError("expected three input operands");
+  }
+  if (getNumOutputs() != 1) {
+    return op->emitOpError("expected one output operand");
+  }
+  const int64_t keyRank = getKeyRank();
+  if (keyRank != 3) {
+    return op->emitError("expected key tensor to have rank 3");
+  }
+  const ShapedType queryType = getQueryType();
+  const ShapedType keyType = getKeyType();
+  const ShapedType valueType = getValueType();
```

Comment:
Nit: we don't use this kind of const in LLVM/MLIR, the code in iree-dialect is intended for upstreaming, so it's better to align on that.

Reply:
We dont use `const` in IREE as well.